pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/SourceDistTask.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.tasks
+
+import groovy.transform.CompileStatic
+import org.gradle.api.tasks.OutputFile
+import org.gradle.process.ExecResult
+
+@CompileStatic
+public class SourceDistTask extends AbstractPythonMainSourceDefaultTask {
```

Comment:
Please leave this as a Java class. This will allow us to move to Java 8 in the future.